.github/workflows/ci.yaml

```
@@ -107,11 +107,79 @@ env:
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+
+      - name: git add
+        run: git add -u
+
+      - name: git commit
+        id: git-commit
+        run: git commit -m "run `cargo fmt --all`"
+        continue-on-error: true # Will fail if nothing to commit
+
+      - name: Comment on PR
+        if: ${{ steps.git_commit.outcome == 'success' }}
```

actionlint (1.7.8)

property "git_commit" is not defined in object type {git-commit: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)